dashboard/test/controllers/admin_nps_controller_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class AdminNpsControllerTest < ActionController::TestCase
+  include Devise::Test::ControllerHelpers
+
+  setup do
+    @admin = create(:admin)
+    @not_admin = create(:teacher, username: 'notadmin', email: 'not_admin@email.xx')
```

Comment:
nit: to make it extra super clear, can you add `admin: false` to this?

Reply:
Ah yes good call. Done!